### DISCUSSION TITLE
Love the new HTML button! And a request...

### DISCUSSION THREAD

**Discussion by ManOrMonster** [BlafKing/sd-civitai-browser-plus#186] [Ideas]
With this new HTML Civitai info, I can finally stop using Civitai Helper. I'd been using it all this time to pull down json with generation info so that it could be put in the prompt with one click.

Could we get a second button that sends only the prompt itself to txt2img? It's nice to send the exact generation info, but most of the time I would prefer to only use the prompt itself.

**Comment by BlafKing** [comment]
Sounds good, that should definitely be possible!

I'll make it so that the user can press any of the individual main metadata options to insert them into txt2img!

**Comment by BlafKing** [comment]
This has now been added in the latest verison! (v3.3.1)
